### PR TITLE
Change documentation for restart app method

### PR DESCRIPTION
**Pull request by marcopollivier**
Adding swagger definition for app restart service v1.0 

Issue #1992

### CODE REVIEW

File: docs/reference/api.yaml [hunk 1]
```diff
@@ -620,6 +620,52 @@ paths:
           description: App not found
           schema:
             $ref: "#/definitions/ErrorMessage"
+
+# title: app restart
```
Comment (andrestc) [inline docs/reference/api.yaml]: These comments are not really necessary.

File: docs/reference/api.yaml [hunk 2]
```diff
@@ -620,6 +620,52 @@ paths:
...
+      description: Restart App.
+      tags:
+        - app
+      security:
+        - Bearer: []
+      consumes:
+        - application/x-www-form-urlencoded
+      parameters:
+        - name: process
+          in: process
```
Comment (andrestc) [inline docs/reference/api.yaml]: This should be `in: formData`

File: docs/reference/api.yaml [hunk 3]
```diff
@@ -620,6 +620,52 @@ paths:
...
+          type: number
+          required: true
```
Comment (andrestc) [inline docs/reference/api.yaml]: The `process` field should be a `string` and its **not** required (a user may request that the whole app be restarted).

**Comment by andrestc** [comment]
Thanks for contributing! Left a few comments about minor issues.

**Comment by marcopollivier** [comment]
Thank you, @andrestc . I will fix it soon.

**Comment by marcopollivier** [comment]
@andrestc Fixed 

image

**Comment by marcopollivier** [comment]
@andrestc Fixed =)


image

**Pull request merged**